### PR TITLE
feat: Add getFormSubmission to script functions

### PR DESCRIPTION
**Pull request by rstyan**

### CODE REVIEW

File: apps/script-service/Services/IScriptFunctions.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ interface IScriptFunctions
 
   string? GeneratePdf(string templateId, string filename, object values);
   IDictionary<string, object?>? GetConfiguration(string @namespace, string name);
-  FormDataResult? GetFormData(string formId);
```
Comment (jonathanweyermann) [inline apps/script-service/Services/IScriptFunctions.cs:10]: Are you getting rid of an existing functionality here ?

File: apps/script-service/Services/ScriptFunctions.cs [hunk 2]
```diff
@@ -73,17 +73,17 @@ public ScriptFunctions(AdspId tenantId, IServiceDirectory directory, Func<Task<s
     return result?.Configuration;
   }
 
-  public FormDataResult? GetFormData(string formId)
+  public virtual FormSubmissionResult? GetFormSubmission(string formId, string submissionId)
```
Comment (jonathanweyermann) [inline apps/script-service/Services/ScriptFunctions.cs:76]: Are you getting rid of an existing functionality here as well?

Also what does virtual do in this case?

**Review by jonathanweyermann — CHANGES_REQUESTED** [review]

File: apps/script-service/Services/IScriptFunctions.cs (hunk 1 above)
Comment (rstyan) [reply apps/script-service/Services/IScriptFunctions.cs:10]: Good catch.  Musta been brain dead that day.

File: apps/script-service/Services/ScriptFunctions.cs (hunk 2 above)
Comment (rstyan) [reply apps/script-service/Services/ScriptFunctions.cs]: No, that was a copy-paste problem.  Another good catch.

Comment (rstyan) [reply apps/script-service/Services/ScriptFunctions.cs:90]: Actually, it is needed for the override in StubScriptFunctions.

**Review by jonathanweyermann — APPROVED** [review]

**Pull request merged**